components/learn/ExternalRecommendedReadings.vue

```diff
@@ -21,8 +21,15 @@
           :segment="link.segment"
           :url="link.url"
         >
-          {{ link.label }}
+          <span v-if="link.author">
+            {{ link.author }}
+          </span>
+          <span :class="link.author ? 'external-recommended-readings__title' : ''">
+            {{ link.label }}
+          </span>
         </AppLink>
+        {{ link.description ? '-' : '' }}
```

Comment:
For these two lines, you should be able to refactor into something like so:

```html
<span v-if="link.description"> - {{ link.description }}</span>
```

Reply:
Cool, thanks